Patches/Core/FactionDefs/Factions_Misc.xml

```diff
@@ -26,73 +26,102 @@
 -->
 
   <Operation Class="PatchOperationRemove">
-  	<xpath>*/FactionDef[defName="OutlanderCivil" or defName="Pirate"]/pawnGroupMakers/li/*/GrenadierEMP</xpath>
+  	<xpath>*/FactionDef[defName="OutlanderCivil" or defName="Pirate"]/pawnGroupMakers/li/*/Grenadier_EMP</xpath>
   </Operation>
 
   <!-- ========== Add tribal grenadies ========== -->
 
   <Operation Class="PatchOperationAdd">
-    <xpath>*/FactionDef[defName="TribeCivil"]/pawnGroupMakers/li[1]/options</xpath>
+    <xpath>*/FactionDef[@Name="TribeBase"]/pawnGroupMakers/li[1]/options</xpath>
     <value>
-      <TribalGrenadier>20</TribalGrenadier>
-      <TribalGunner>20</TribalGunner>
+      <Tribal_Grenadier>5</Tribal_Grenadier>
```

Comment:
Why the weight changes?

Reply:
To have it line up with the vanilla weights. I haven't seen what they were before but assume they are much lower. Vanilla weights are between 1 and 10 for any type.

Reply:
Should double-check to ensure proportionality here.